app/models/review.rb

```diff
@@ -0,0 +1,13 @@
+class Review < ApplicationRecord
+  after_create :update_average_rating
+  belongs_to :restaurant
+
+  validates :reviewer, :rating, :comment, presence: true
+  validates :rating, inclusion: { in: [0, 1, 2, 3] }
+
+  def update_average_rating
+    reviews_metadata = restaurant.restaurant_reviews_metadatum
+    avarageRating = (reviews_metadata.reviews_count * reviews_metadata.avarage_score + rating) / (reviews_metadata.reviews_count + 1)
```

Comment:
how do you handle floats here?

```
irb(main):004:0> (5 * 3 + 2) / 11
=> 1
irb(main):005:0> (5 * 3 + 2.0) / 11
=> 1.5454545454545454
```

Reply:
@yanivmichaelis this is a UI concern and doesn't need to be handled here. **Maybe** in serializer, but probably in the client - i.e. React or a rails helper if using ERB